docs/gmrq.rst

```diff
@@ -1,8 +1,33 @@
 .. _gmrq:
 
-Model Selection (GMRQ)
+Model Selection using GMRQ
 ======================
 
-GMRQ uses a variational principle to help you select model parameters.
+The generalized matrix Rayleigh quotient (GMRQ) is a specific application of
+the variational principle (adapted from `quantum mechanics <https://en.wikipedia.org/wiki/Variational_method_(quantum_mechanics)>_`)
```

Comment:
The github render shows it weird. Maybe theres not supposed to be a space before `<`?